addons/longhorn/template/base/install.sh

```diff
@@ -119,3 +122,33 @@ function longhorn_preflight() {
     local src="${DIR}/addons/longhorn/${LONGHORN_VERSION}"
     echo "${src}/host-preflight.yaml"
 }
+
+function check_mount_propagation() {
+    local src=$1
+
+    kubectl get ns longhorn-system >/dev/null 2>&1 || kubectl create ns longhorn-system >/dev/null 2>&1
+
+    render_yaml_file "$src/tmpl-mount-propagation.yaml" > "$src/mount-propagation.yaml"
+    kubectl apply -f "$src/mount-propagation.yaml"
```

Comment:
what if this daemonset existed prior to whatever changes were applied?

Reply:
I'll add a delete command first then